src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -82,6 +82,12 @@
   <module name="org.openhab.tools.analysis.checkstyle.ImportExportedPackagesCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.ManifestPackageVersionCheck">
+     <property name="severity" value="warning" />
+     <property name="ignoreImportedPackages" value="org.junit.*,
```

Comment:
I do not care too much about the test fragments, so should only have loose checks on them - so yes, for me it is ok to ignore it there.